posthog/api/team.py

```diff
@@ -95,6 +96,8 @@ def validate(self, attrs: Any) -> Any:
         if "primary_dashboard" in attrs and attrs["primary_dashboard"].team != self.instance:
             raise exceptions.PermissionDenied("Dashboard does not belong to this team.")
 
+        # TODO: Add validation here for person_display_name_properties to be list of strings?
```

Comment:
I think we should make the field in the db a postgres array of strings. That'll solve most issues for free, and we'll convert this to a model if it needs to be more complex. So no real need for JSON.